enums/uncefactMassOrVolumeUnits.json

```diff
@@ -3,13 +3,13 @@
     "anyOf":[
        {
           "$ref": "../enums/uncefactMassUnits.json",
-          "nullable": true,
+          "type": ["string", "null"],
```

Comment:
You cannot specify "type" if using "$ref". The type should be edited inside the referenced enumeration file. You will need to check all other uses of the referenced file to ensure it is ok for the value to be nullable everywhere it is used.